doc/man1/openssl-cms.pod.in

```diff
@@ -82,7 +82,7 @@ B<openssl> B<cms>
 {- $OpenSSL::safe::opt_r_synopsis -}
 {- $OpenSSL::safe::opt_engine_synopsis -}
 {- $OpenSSL::safe::opt_provider_synopsis -}
-[I<cert.pem> ...]
+[I<recipcert> ...]
```

Comment:
I misread that as "recipient" several times over.  May I suggest `I<recipient-cert>`?

Reply:
Done